### ISSUE TITLE
Nondispelable Debuffs being hidden 

### ISSUE THREAD

**Issue by Kalply** [jordonwow/bigdebuffs#443]
After the latest update certain non dispelable debuffs are being hidden by the addon. Zap on the last boss in mechagon workshop, the bleeds in iron docks, etc.

**Comment by kinarpt** [comment]
Also grevious wounds stacks don't show , I can imagine necrotic won't either.

**Comment by pedroSG94** [comment]
I have the same error

**Comment by cleanlock** [comment]
yup, same here. also doesn't show the icon in my unitframe (Pala Wings for example)
